tic-tac-toe/src/urbit/mar/toe/player.hoon

```diff
@@ -0,0 +1,13 @@
+::
+::::  /hoon/player/toe/mar
+  ::
+/?    310
+/-    toe
+=,    toe
+!:
+::::  A minimal player mark
+|_  toe-player
+++  grab  |%
+          ++  noun  toe-player
+          --
```

Comment:
Can you move this to under the `++  grab` in all your marks? This looks super funky. (;